mock/datasets/no2.data.mdx

```diff
@@ -55,6 +55,25 @@ layers:
       id: polarNorth
     bounds: [-10, 36, -5, 42]
     description: Levels in 10¹⁵ molecules cm⁻². Darker colors indicate higher nitrogen dioxide (NO₂) levels associated and more activity. Lighter colors indicate lower levels of NO₂ and less activity.
+    info:
+      modal:
+        subtitle: |
+          ::markdown
+          Global, monthly 1 km resolution dataset of fossil fuel carbon dioxide emissions, version 2022
+        contents: |
+          ::markdown
+          - Temporal Extent: January 2000 - December 2021
+          - Temporal Resolution: Monthly
+          - Spatial Extent: Global
+          - Spatial Resolution: 1 km x 1 km
+          - Data Units: Tons of carbon per 1 km x 1 km cell (monthly total)
+          - Data Type: Research
+          - Data Latency: Updated annually, following the release of an updated [BP Statistical Review of World Energy report](https://www.bp.com/en/global/corporate/energy-economics.html)
```

Comment:
Yeah, I agree. Especially since our current application also asks for structured data (why `info` has a separate section for `template`). We would need more streamlined thoughts about how to handle this layer information before though. I can't think of a straightforward way of having both descriptive layerinfo and structured layerinfo without manual edits and duplication of the data.